spec/etapi/notes.spec.ts

```diff
@@ -1,107 +1,5 @@
-import crypto = require("crypto");
-import etapi = require("../support/etapi");
+describe("Notes", () => {
```

Comment:
This is weird, but it's what upstream has. The original spec file is still maintained, but in `notes.ts` (thus not being executed, see #214).